app/Utils/ApplicationHandler.php

```diff
@@ -20,8 +20,9 @@ public function storeQuestionsData(Request $request, User $user): void
     {
         $data = $request->validate([
             'status' => 'required|in:extern,resident',
-            'graduation_average' => 'required|numeric',
-            'semester_average' => 'nullable',
+            'graduation_average' => 'required|numeric|min:1|max:5',
+            'semester_average' => 'nullable|array',
+            'semester_average.*' => 'numeric|min:1|max:5',
```

Comment:
The thing here is that if its from another country, it might not fall between 1-5.

Reply:
Oh, I see... I'll revert that then.